src/include/platform/ConnectivityManager.h

```diff
@@ -188,6 +193,13 @@ class ConnectivityManager
     CHIP_ERROR GetNetworkInterfaces(NetworkInterface ** netifpp);
     void ReleaseNetworkInterfaces(NetworkInterface * netifp);
 
+// Sleepy end device methods
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+    CHIP_ERROR GetSEDPollingConfig(SEDPollingConfig & pollingConfig);
+    CHIP_ERROR SetSEDPollingConfig(const SEDPollingConfig & pollingConfig);
+    CHIP_ERROR RequestSEDFastPollingMode(bool onOff);
```

Comment:
It would be good to document the semantics of these, esp the third one, which means something pretty specific by "off"...